components/builder-web/app/dashboard/_dashboard.component.scss

```diff
@@ -157,8 +157,8 @@
             }
 
             hab-icon {
-              width: 24px;
-              height: 24px;
+              width: 20px;
+              height: 20px;
```

Comment:
Seems like this should be a var maybe?

Reply:
Yep, good idea. There's a lot we can do to refactor and reduce CSS duplication in general. Might as well start now! 😄